src/WPF/ArcGISRuntime.WPF.Viewer/Samples/UtilityNetwork/DisplayUtilityNetworkContainer/DisplayUtilityNetworkContainer.xaml.cs

```diff
@@ -0,0 +1,225 @@
+
+        // This viewpoint shows several associations clearly in the utility network.
+        private readonly Viewpoint InitialViewpoint = new Viewpoint(41.801504, -88.163718, 4e3);
+
+        // Utility network that will be created from the feature server.
+        private UtilityNetwork _utilityNetwork;
+
+        public DisplayUtilityNetworkContainer()
+        {
+            InitializeComponent();
+            _ = Initialize();
+        }
+
+        private async Task Initialize()
+        {
+            // As of ArcGIS Enterprise 10.8.1, using utility network functionality requires a licensed user. The following login for the sample server is licensed to perform utility network operations.
+            AuthenticationManager.Current.ChallengeHandler = new ChallengeHandler(async (info) =>
+            {
+                try
+                {
+                    // WARNING: Never hardcode login information in a production application. This is done solely for the sake of the sample.
+                    string sampleServer7User = "viewer01";
+                    string sampleServer7Pass = "I68VGU^nMurF";
+
+                    return await AuthenticationManager.Current.GenerateCredentialAsync(info.ServiceUri, sampleServer7User, sampleServer7Pass);
+                }
+                catch (Exception ex)
+                {
+                    Debug.WriteLine(ex.Message);
+                    return null;
+                }
+            });
+
+            // Create a graphics overlay for associations.
+            _associationsOverlay = new GraphicsOverlay();
+            MyMapView.GraphicsOverlays.Add(_associationsOverlay);
+
+            // Symbols for the associations.
+            Symbol attachmentSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Dot, Color.Blue, 5d);
+            Symbol boundingBoxSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Dash, Color.Yellow, 5d);
+            Symbol connectivitySymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Dot, Color.Red, 5d);
+
+            // Create a renderer for the associations.
+            var attachmentValue = new UniqueValue("Attachment", string.Empty, attachmentSymbol, UtilityAssociationType.Attachment.ToString());
+            var boundingBoxValue = new UniqueValue("Containment", string.Empty, boundingBoxSymbol, UtilityAssociationType.Containment.ToString());
+            var connectivityValue = new UniqueValue("Connectivity", string.Empty, connectivitySymbol, UtilityAssociationType.Connectivity.ToString());
+            _associationsOverlay.Renderer = new UniqueValueRenderer(new List<string> { "AssociationType" }, new List<UniqueValue> { attachmentValue, boundingBoxValue, connectivityValue }, string.Empty, null);
+
+            // Populate the legend in the UI.
+            Dictionary<UtilityAssociationType, System.Windows.Media.ImageSource> legend;
+            legend = new Dictionary<UtilityAssociationType, System.Windows.Media.ImageSource>();
```

Comment:
This should be collapsed into one line; I think `var` is fine here since the type name is so long.

Separately, I recommend renaming this, to avoid confusion with a proper legend; that has a specific meaning within the platform, and it might be confusing when we have e.g. the legend toolkit component.

I think `key` would be a good starting point for coming up with another name.